composition/launch/composition_demo.launch.py

```diff
@@ -37,8 +35,8 @@ def generate_launch_description():
                     package='composition',
                     node_plugin='composition::Listener',
                     node_name='listener')
-            ]
+            ],
+            output='screen',
```

Comment:
I assume printing to screen is what you want by default? Maybe not, since you can force it out manually with `ros2 launch -a`?

Reply:
Yes, printing as default is what I was going for.